### PR TITLE
feat(): added new endpoints

### PR DESCRIPTION
**Pull request by JJ-Cro**
Added all missing endpoints from recent updates
Added examples for Bybit website

### CODE REVIEW

File: src/rest-client-v5.ts [hunk 1]
```diff
@@ -2167,4 +2171,37 @@ export class RestClientV5 extends BaseRestClient {
...
+
+  /**
+   * Issue a voucher to a user
+   *
+   * INFO
+   * Use exchange broker master account to issue
+   */
+  issueBrokerVoucher(
+    params: IssueVoucherParamsV5,
+  ): Promise<APIResponseV3<any>> {
```
Comment (tiagosiebler) [inline src/rest-client-v5.ts]: No response type?

File: src/rest-client-v5.ts [hunk 2]
```diff
@@ -2167,4 +2171,37 @@ export class RestClientV5 extends BaseRestClient {
       params,
     );
   }
+
+  /**
+   * Query Voucher Spec
+   */
+  getBrokerVoucherSpec(params: {
+    id: string;
+  }): Promise<APIResponseV3WithTime<BrokerVoucherSpec>> {
```
Comment (tiagosiebler) [inline src/rest-client-v5.ts]: Suffix the type with "V5" please

**Review by tiagosiebler — CHANGES_REQUESTED** [review]

File: src/rest-client-v5.ts (hunk 1 above)
Comment (JJ-Cro) [reply src/rest-client-v5.ts]: No, returns empty object

File: src/rest-client-v5.ts (hunk 2 above)
Comment (JJ-Cro) [reply src/rest-client-v5.ts]: Done

**Review by tiagosiebler — APPROVED** [review]
LGTM, thanks

**Pull request merged**